build.gradle

```diff
@@ -266,6 +164,113 @@ subprojects {
   }
 }
 
+def pomFiller(pom, nameVar, descriptionVar) {
+  pom.project {
+    name = nameVar
+    description = descriptionVar
+    url = 'http://ehcache.org'
+    organization {
+      name = 'Terracotta Inc., a wholly-owned subsidiary of Software AG USA, Inc.'
+      url = 'http://terracotta.org'
+    }
+    issueManagement {
+      system = 'Github'
+      url = 'https://github.com/ehcache/ehcache3/issues'
+    }
+    scm {
+      url = 'https://github.com/ehcache/ehcache3'
+      connection = 'scm:git:https://github.com/ehcache/ehcache3.git'
+      developerConnection = 'scm:git:git@github.com:ehcache/ehcache3.git'
+    }
+    licenses {
+      license {
+        name = 'The Apache Software License, Version 2.0'
+        url = 'http://www.apache.org/licenses/LICENSE-2.0.txt'
+        distribution = 'repo'
+      }
+    }
+    developers {
```

Comment:
The reason I moved this block from the top to the bottom of the file is that it is code that needs little attention while all the basic version and plugins info is really more important and now shows properly at the top (copyright header excepted)